using-mobile-extensions/adobe-journey-optimizer/api-reference.md

```diff
@@ -203,125 +218,134 @@ The setPushIdentifier API sets the push token, allowing you to sync it with Prof
 
-### Swift
+#### Swift
 
-#### Syntax
+**Syntax**
 
 ```swift
 public static func setPushIdentifier(_ deviceToken: Data?)
 ```
 
-| **Variable** | **Type** | **Description** |
+| Variable | Type | Description |
 | :----------- | :------- | :-------------- |
 | `deviceToken` | Data | The push token that is synced with Adobe Experience Platform. |
 
-#### Example
+**Example**
 
 ```swift
 func application(_: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
     MobileCore.setPushIdentifier(deviceToken)
 }
 ```
 
-### Objective-C
+#### Objective-C
 
-#### Syntax
+**Syntax**
 
 ```objc
 public static func setPushIdentifier(_ deviceToken: Data?)
 ```
 
-| **Variable** | **Type** | **Description** |
+| Variable | Type | Description |
 | :----------- | :------- | :-------------- |
 | `deviceToken` | Data | The push token that is synced with Adobe Experience Platform. |
 
-#### Example
+**Example**
 
 ```objc
-- (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken { 
+- (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
     [AEPMobileCore setPushIdentifier:deviceToken];
 }
 ```
+
 {% endtab %}
+
 {% endtabs %}
 
-## addPushTrackingDetails <a id="addpushtrackingdetails"></a>
+## addPushTrackingDetails (Android only) <a id="addpushtrackingdetails"></a>
 
-The addPushTrackingDetails API is used to update a pending intent with important information, such as messageId and Customer Journey information. 
+The `addPushTrackingDetails` API is used to update a pending intent with important information, such as messageId and Customer Journey information.
 
 **Note:** Calling this API is mandatory, so the pending intent can be used while tracking push notification interactions.
 
 {% tabs %}
+
 {% tab title="Android" %}
-### Java
 
-#### Syntax
+#### Java
+
+**Syntax**
 
 ```java
 public static boolean addPushTrackingDetails(final Intent intent, final String messageId, final Map<String, String> data)
 ```
 
-| **Variable** | **Type** | **Description** |
+| Variable | Type | Description |
 | :----------- | :------- | :-------------- |
 | `intent` | `Intent` | The pending intent that needs to be updated so it can be used when the user interacts with the notification. |
 | `messageId` | `String` | The message ID for the push notification. |
 | `data` | `Map<String, String>` | The data of the remoteMessage. |
 
 This API returns a boolean, indicating whether the intent was updated with necessary information (messageId and Customer Journey data).
 
-#### Example
+**Example**
 
 ```java
 boolean success = addPushTrackingDetails(intent, messageId, data)
 ```
+
 {% endtab %}
+
 {% endtabs %}
 
 ## Public classes
 
-### MessagingPushPayload
+### MessagingPushPayload (Android only)
```

Comment:
can we make the h2 to match the other API headings?

Reply:
realized its a class description. should we make it h2 to match? @calebk1 what do you think?

Reply:
Hmm, since it's a sub-header off of public classes, having it as an H3 should be fine.